### PR TITLE
Feature/requirements

### LINKED ISSUE #6: Add requirements info about CPU and memory
As explained [here](https://github.com/ufcg-lsd/saps-scheduler/issues/5), SAPS should send CPU and memory information for better job scheduling based on required resources. In this repository there is a [ExecutionScriptTag class](https://github.com/ufcg-lsd/saps-common/blob/develop/src/main/java/saps/common/utils/ExecutionScriptTag.java) that deals with the information in the scripts and that must be modified to keep this data.

### PR DESCRIPTION
**Pull request by thiagoyeds**
Fixed #6

### CODE REVIEW

File: src/main/java/saps/common/utils/ExecutionScriptTag.java [hunk 1]
```diff
@@ -10,12 +10,17 @@
...
+	private String cpuUsage;
 
-	public ExecutionScriptTag(String name, String dockerRepository, String dockerTag, String type) {
+	public ExecutionScriptTag(String name, String dockerRepository, String dockerTag, String memoryUsage,
+			String cpuUsage, String type) {
 		this.name = name;
 		this.dockerRepository = dockerRepository;
 		this.dockerTag = dockerTag;
 		this.type = type;
+		this.memoryUsage = memoryUsage;
```
Comment (thiagomanel) [inline src/main/java/saps/common/utils/ExecutionScriptTag.java]: I'm missing the arg checks on this constructor (i'm afraid we have this problem spread on the whole codebase). As these parameters have some sort of defined pattern (16g, g means gigabytes and so on) we need to check the format.

In addition to that, Usage does not seem a good name. Is it a requirement, correct?

**Review by thiagomanel — COMMENTED** [review]
a request to make the code safer

File: src/main/java/saps/common/utils/ExecutionScriptTag.java (hunk 1 above)
Comment (thiagoyeds) [reply src/main/java/saps/common/utils/ExecutionScriptTag.java]: Correct, I made some changes, review again

**Review by thiagomanel — CHANGES_REQUESTED** [review]
add an unit test

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**